src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -0,0 +1,237 @@
+      </description>
+      <arg name="Status" type="ENUM8"/>
+      <arg name="GroupId" type="group_id"/>
+      <arg name="SceneId" type="INT8U"/>
+    </command>
+    
+    <command source="server" code="0x03" name="RemoveAllScenesResponse" optional="false" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast RemoveAllScenes command
+      </description>
+      <arg name="Status" type="ENUM8"/>
+      <arg name="GroupId" type="group_id"/>
+    </command>
+    
+    <command source="server" code="0x04" name="StoreSceneResponse" optional="false" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast StoreScene command
+      </description>
+      <arg name="Status" type="ENUM8"/>
+      <arg name="GroupId" type="group_id"/>
+      <arg name="SceneId" type="INT8U"/>
+    </command>
+    
+    <command source="server" code="0x06" name="GetSceneMembershipResponse" optional="false" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast GetSceneMembership command
+      </description>
+      <arg name="Status" type="ENUM8"/>
+      <arg name="Capacity" type="INT8U" isNullable="true"/>
+      <arg name="GroupId" type="group_id"/>
+      <arg name="SceneList" type="INT8U" array="true" optional="true"/>
+    </command>
+
+  <command source="server" code="0x40" name="EnhancedAddSceneResponse" optional="true" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast EnhancedAddScene command
+      </description>
+      <arg name="Status" type="ENUM8"/>
+      <arg name="GroupId" type="group_id"/>
+      <arg name="SceneId" type="INT8U"/>
+    </command>
+
+    <command source="server" code="0x41" name="EnhancedViewSceneResponse" optional="true" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast EnhancedViewScene command
+      </description>
+      <arg name="Status" type="ENUM8"/>
+      <arg name="GroupId" type="group_Id"/>
+      <arg name="SceneId" type="INT8U"/>
+      <arg name="TransitionTime" type="INT16U"  optional="true"/>
+      <arg name="SceneName" type="CHAR_STRING"  optional="true"/>
+      <arg name="ExtensionFieldSets" type="ExtensionFieldSet" array="true" optional="true"/>
+    </command>
+
+    <command source="server" code="0x42" name="CopySceneResponse" optional="true" disableDefaultResponse="true">
+      <description>
+        The command is generated in response to a received unicast CopyScene command
+      </description>
+      <arg name="Status" type="ENUM8"/>
+      <arg name="GroupIdFrom" type="group_Id"/>
+      <arg name="SceneIdFrom" type="INT8U"/>
+    </command>
+  </cluster>
+
+  <bitmap name="SceneFeatures" type="BITMAP32">
```

Comment:
Do we have a standardized way of defining features? This ties into my comment [here](https://github.com/project-chip/connectedhomeip/pull/18667#discussion_r878335889) for a scalable solution to emitting pre-processor defines for feature flags that can be used by the scenes (and other) cluster impls that need such mechanisms.

Reply:
This is the way I have seen the definition of the features bit map defines across other cluster definitions in Matter. I don't if there is a standardized way mentioned anywhere.

Maybe someone else knows more about this